pyvista/core/filters/image_data.py

```diff
@@ -1226,6 +1235,14 @@ def contour_labels(  # type: ignore[misc]
         using ``select_inputs`` converts previously-internal boundaries into external
         ones.
 
+        Disable the generation of a closed surface. Since the input image has
+        foreground regions visible at the edges of the image (e.g. the ``+Z`` bound),
+        setting ``closed_surface=False`` in this example causes the top and sides of
+        the mesh to be "open".
+
+        >>> surf = image.contour_labels(closed_surface=False)
+        >>> surf.plot(zoom=1.5, **plot_kwargs)
```

Comment:
```suggestion
        Do not pad the image with background values before contouring. Since the input image 
        has foreground regions visible at the edges of the image (e.g. the ``+Z`` bound),
        setting ``pad_background=False`` in this example causes the top and sides of
        the mesh to be "open".

        >>> surf = image.contour_labels(pad_background=False)
        >>> surf.plot(zoom=1.5, **plot_kwargs)
```